GoogleForCreators/web-stories-wp
# Added tooltips for border toggles

## Context

Added tooltips. Wording to be discussed.

## User-facing changes

Note: the tooltips have since been changed to:

* _Toggle consistent border_
* _Toggle consistent corner radius_

| Before | After |
|-|-|
| ![Screen Shot 2021-05-04 at 17 29 43](https://user-images.githubusercontent.com/637548/117072465-7afcf180-acfe-11eb-8d24-67759fefc6e4.png) | ![Screen Shot 2021-05-04 at 17 25 39](https://user-images.githubusercontent.com/637548/117072472-7cc6b500-acfe-11eb-92c5-ca7524511ee3.png) |
| ![Screen Shot 2021-05-04 at 17 29 47](https://user-images.githubusercontent.com/637548/117072475-7df7e200-acfe-11eb-99a0-69247a706f08.png) | ![Screen Shot 2021-05-04 at 17 25 35](https://user-images.githubusercontent.com/637548/117072481-7fc1a580-acfe-11eb-88d2-152e083e17f4.png) |

## Testing Instructions



### QA

This PR can be tested by following these steps:

1. Focus the toggle buttons in either the border width panel or corner radius panel
2. Observe that the toggle button has a tooltip

### UAT




## Checklist




---

Fixes #6744
